frontend/src/lib/api/zaakSelection.ts

```diff
@@ -12,6 +12,7 @@ export async function getSelection<DetailType = unknown>(
   signal?: AbortSignal,
 ) {
   const params = new URLSearchParams(exp as Record<string, string>);
+  // eslint-disable-next-line @typescript-eslint/no-unused-expressions
   selectedOnly && params.set("selected", "true");
```

Comment:
I have... opinions :D

I think the linter rule points out a code smell here and would suggest using a simple `if (selectedOnly) ... ` block since you're imperatively performing mutation here. took me a minute to understand what the problem was and what the code was doing.

Reply:
Same as other